Dfe.Academies.Academisation.Service.UnitTest/Commands/ApplicationDeleteCommandTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using AutoFixture;
+using Dfe.Academies.Academisation.Core;
+using Dfe.Academies.Academisation.Domain.ApplicationAggregate;
+using Dfe.Academies.Academisation.Domain.Core.ApplicationAggregate;
+using Dfe.Academies.Academisation.Domain.SeedWork;
+using Dfe.Academies.Academisation.IData.ProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.ApplicationAggregate;
+using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.Services;
+using Dfe.Academies.Academisation.IService.RequestModels;
+using Dfe.Academies.Academisation.Service.Commands.Application;
+using Moq;
+using Xunit;
+
+namespace Dfe.Academies.Academisation.Service.UnitTest.Commands
+{
+	public class ApplicationDeleteCommandTests
+	{
+		private readonly Fixture _fixture = new();
+
+		private readonly Mock<IApplication> _applicationMock = new();
+		private readonly Mock<IApplication> _applicationMock1 = new();
+
+		private readonly Mock<IProject> _projectMock = new();
```

Comment:
The _applicationMock1, _projectMock and _projectCreateDataCommand Mock objects are created but not used in the tests, this might be redundant.